assets/source/sass/frontend/pinterest-for-woocommerce-pins.scss

```diff
@@ -24,10 +24,12 @@
 
 
 // Twenty Twenty-Four compatibility.
-.wp-block-post {
+.wp-block-product {
```

Comment:
I believe you mean this?

```suggestion
.wc-block-product {
```

Reply:
@message-dimke , do you happen to know when and where `wp-block-post` is changed to `wc-block-product`? I was searching around but couldn't find the info.

Also, maybe for backward compatibility, we need to have both, like the following? (need to test this out.)

```suggestion
.wp-block-post,
.wc-block-product {
```